DependencyInjection/Configuration.php

```diff
@@ -15,8 +16,14 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('limenius_react');
+        if (Kernel::VERSION_ID >= 40200) {
```

Comment:
A simpler approach (adopted by the symfony community) would be:
```php
$treeBuilder = new TreeBuilder('limenius_react');

if (method_exists($treeBuilder, 'getRootNode')) {
    $rootNode = $treeBuilder->getRootNode();
} else {
    // BC layer for symfony/config 4.1 and older
    $rootNode = $treeBuilder->root('limenius_react');
}
```

Reply:
yes, but when I opened this PR, my suggested way was used in other bundles. as there were no activity here, i didn't update this PR to reflect current way of doing things

Reply:
Maybe @nacmartin could merge this one and publish a new release, let's ping him ;)